bot.py

```diff
@@ -35,30 +38,35 @@ async def on_message(self, message):
         Login and loading handling
         """
         if message.guild not in self.clients:
-            self.clients[message.guild] = MusicBot(message.guild)
-        await self.clients[message.guild].on_message(message)
+            self.clients[message.guild] = MusicBot(message.guild, self)
+        await self.clients[message.guild].handle_message(message)
 
 
-class MusicBot(discord.Client):
+class MusicBot:
     """
     The main bot functionality
     """
 
     # pylint: disable=no-self-use
     # pylint: disable=too-many-instance-attributes
 
-    COMMAND_PREFIX = "!"
-    guild = None
-    handlers = None
-    voice_client = None
-    song_queue = None
-    current = None
-    last_text_channel = None
+    COMMAND_PREFIX = "#"
```

Comment:
```suggestion
    COMMAND_PREFIX = "!"
```
We will have to include a test that checks if the COMMAND_PREFIX is actually "!", since we are always changing it while testing 😏